backend/handler/templates/types.ts

```diff
@@ -71,13 +71,13 @@ export interface UpdateTemplateData extends CreateTemplateData {
     oldId: string;
 }
 
-export interface PageGroupFileContent extends CommonFileContent {
+export interface PageGroupFileContent<I = string> extends CommonFileContent {
```

Comment:
```suggestion
export interface PageGroupFileContent<Icon = string> extends CommonFileContent {
```
Applies to other occurrences in this file as well.

Reply:
Changed.